### PR TITLE
set up divorce cfs security scan

### PR DESCRIPTION
**Pull request by MOsinloye**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 
 properties([
-        [$class: 'GithubProjectProperty', projectUrlStr: 'https://github.com/hmcts/div-case-formatter-service'],
-        pipelineTriggers([[$class: 'GitHubPushTrigger']])
-])
+        pipelineTriggers([cron('15 01 * * *')]),
+        parameters([
+                string(name: 'URL_TO_TEST', defaultValue: 'http://div-cfs-aat.service.core-compute-aat.internal', description: 'The URL you want to run these tests against'),
+                string(name: 'SecurityRules',
+                        defaultValue: 'https://gist.githubusercontent.com/satyachundur/0f16e35aedd7c939b33f071f5c447125/raw',
```
Comment (vianvi) [inline Jenkinsfile_nightly]: it shouldn't point to personnel repos

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -1,9 +1,14 @@
 #!groovy
 
 properties([
-        [$class: 'GithubProjectProperty', projectUrlStr: 'https://github.com/hmcts/div-case-formatter-service'],
-        pipelineTriggers([[$class: 'GitHubPushTrigger']])
-])
+        pipelineTriggers([cron('15 01 * * *')]),
+        parameters([
+                string(name: 'URL_TO_TEST', defaultValue: 'http://div-cfs-aat.service.core-compute-aat.internal', description: 'The URL you want to run these tests against'),
```
Comment (vianvi) [inline Jenkinsfile_nightly]: change this to HTTPS

File: security.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+echo ${TEST_URL}
+zap-api-scan.py -t ${TEST_URL}/v2/api-docs -f openapi -u ${SecurityRules} -P 1001
+cat zap.out
+zap-cli --zap-url http://0.0.0.0 -p 1001 report -o /zap/api-report.html -f html
+cp /zap/api-report.html functional-output/
+zap-cli -p 1001 alerts -l Informational
```
Comment (vianvi) [inline security.sh]: i think we agreed we use high

**Comment by MOsinloye** [comment]
both changes made

**Pull request merged**